### DISCUSSION TITLE
如何打断点调试JS代码

### DISCUSSION THREAD

**Discussion by freeksg66** [Tencent/ScriptX#98] [Q&A] [answered]
在我的c++程序里使用ScriptX运行JS脚本，但是随着JS代码逻辑复杂，只靠console.log调试效率太低。有没有什么办法可以像调试C++代码一样打断点？

**Comment by LanderlYoung** [comment, marked as answer]
这个不在ScriptX的范围内，对于V8等引擎有一套inspect API，你需要接入这套API然后使用chrome等工具debug。需要针对具体引擎具体接入。

如何debug：建议在一个可debug的环境里运行脚本（如chrome，node）
